src/main/java/filter/expression/Qualifier.java

```diff
@@ -136,23 +131,61 @@ public static boolean process(IModel model, FilterExpression expr, TurboIssue is
         return exprWithNormalQualifiers.isSatisfiedBy(model, issue, new MetaQualifierInfo(metaQualifiers));
     }
 
+    /**
+     * Get all milestones that can be aliased with the keyword "current".
+     */
+    private static List<TurboMilestone> getMilestonesWithAlias(IModel model, List<String> repoIds) {
```

Comment:
Maybe we should define a new term? The only explanation I can find from the user guide does not define any term for such milestones.

> current or curr can be used to refer to an open milestone with earliest due date, or an overdue open milestone with open issues if any. curr-2, curr-1, curr+1, curr+2, etc. (no space before or after -/+) can then be used to refer to milestones before or after current, sorted by due date.